src/main.ts

```diff
@@ -0,0 +1,10 @@
+import { queueBroker } from './utils/queueBroker';
```

Comment:
Same here, let's keep this code in `src/index.ts` unless we really need to decouple it.